tests/common/MonoTouch.Dialog/Directory.Build.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Using a Directory.Build.props instead of shared.csproj might trip up some xharness logic: https://github.com/xamarin/xamarin-macios/blob/a8e360d9883bcffd24c8f27258ea119084cf215c/tests/xharness/TestProject.cs#L192-L193 (might be the reason for the failures to run the tests).

Reply:
Trying this out